src/client.ts

```diff
@@ -1196,7 +1194,7 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
     }
 
     if (event.channel && event.type === 'notification.message_new') {
-      this.configs[event.channel.type] = event.channel.config;
+      this._addChannelConfig({ channel: event.channel });
```

Comment:
```suggestion
      // possible @ts-ignore here instead?
      this._addChannelConfig(event);
```
I had to pass a new object as TS somehow doesn't respect `event.channel` nullish check of the condition above and thinks that `event.channel` could be `undefined`.

Reply:
It would probably be okay with it if the check is `event.channel !== undefined` and not the vague truthy check that it does now, right? Or does that give the same result?

Reply:
It seems a bit odd to me to pass the channel to a function that by name expects a channel _config_, but that's not that big a deal